src/assets/i18n/uk.json

```diff
@@ -19,6 +19,33 @@
     "ANALYTICS_FALLBACK": "Якась аналітика тут в майбутньому",
     "SIDENAV": {
       "SLOTS": "Часові Слоти"
+    },
+    "CREATE_TIME_SLOT_CARD": {
+      "START_TIME": "Початковий час",
+      "DURATION": "Тривалість",
+      "ERROR": "Слот закінчиться наступного дня"
+    },
+    "TIME_SLOT": {
+      "LESSON": "Фортепіано",
+      "TEACHER": "Марія Бойко",
```

Comment:
remove

Reply:
Resolved in 71246f23aebad5a25b7068cd1cd89704b5960a38